cmd/xl-storage.go

```diff
@@ -2922,42 +2930,43 @@ func (s *xlStorage) bitrotVerify(ctx context.Context, partPath string, partSize
 	return bitrotVerify(diskHealthReader(ctx, file), fi.Size(), partSize, algo, sum, shardSize)
 }
 
-func (s *xlStorage) VerifyFile(ctx context.Context, volume, path string, fi FileInfo) (err error) {
+func (s *xlStorage) VerifyFile(ctx context.Context, volume, path string, fi FileInfo) ([]error, error) {
```

Comment:
```suggestion
func (s *xlStorage) VerifyFile(ctx context.Context, volume, path string, fi FileInfo) ([]error, error) {
```

Again use a specific response instead of relying on error string parsing.